pkg/promclient/promclient.go

```diff
@@ -121,7 +121,9 @@ func (c *Client) req2xx(ctx context.Context, u *url.URL, method string) (_ []byt
 	if method == http.MethodPost {
 		req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
 	}
-
+	if tenantHeader != "" {
```

Comment:
Are we using any default tenant header, in case `tenantHeader` is empty string?

Reply:
No, we are not using any default tenant header value.
In case tenantHeader is an empty string we assume that the user has access to tenant it specifies.